src/Components/Web/src/Forms/AttributeUtilities.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.AspNetCore.Components.Forms;
 
 internal static class AttributeUtilities
 {
-    public static string CombineClassNames(IReadOnlyDictionary<string, object>? additionalAttributes, string classNames)
+    public static string? CombineClassNames(IReadOnlyDictionary<string, object>? additionalAttributes, string? classNames)
```

Comment:
The nullability changes here and below are just because this code actually does work fine with a null `classNames`, and the new code benefits by not having to do a `?? string.Empty`.